2-promises/solution.js

```diff
@@ -23,17 +23,29 @@ const id = yourRandomMethod() //third run
 6. use promise chaining accordingly, don't forget error handling
 7. log the resultant fullname, or the error, at the end
 */
+const firstNameMethodImport = require("./firstnames")
+const lastNameMethodImport = require("./lastnames")
+
+const ramdonMethodId = () => Math.floor(Math.random() * 101)
```

Comment:
Same here. Please avoid typos and using the string itself `Method` and also set a descriptive/meaningful method name
```suggestion
const getRandomId = () => Math.floor(Math.random() * 101)
```
Also, could you explain to me why you are multiplying the got random value  by  `101` (`Math.floor(Math.random() * 101)`)